test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_performance_tensorflow_sm_training.py

```diff
@@ -19,6 +19,7 @@
 )
 
 
+@pytest.mark.usefixtures("sagemaker_only")
```

Comment:
Generally for test parameterization, we use fixtures (i.e. things like cpu_only, gpu_only, etc). `usefixtures` is functionally equivalent to adding the fixture in as a function argument, but since it isn't used as a variable in the test, it makes more sense to define it outside of the test function.

Markers would require a different implementation, likely with separate pytest executions for SM and DIY. I would prefer to keep it this way since it allows us more flexibility to use `pytest_generate_tests`